language/stdlib/modules/LibraVMConfig.move

```diff
@@ -40,19 +40,23 @@ module LibraVMConfig {
         /// The cost per-byte written to storage.
         global_memory_per_byte_write_cost: u64,
 
-        /// We charge one unit of gas per-byte for the first 600 bytes
+        /// The flat minimum amount of gas required for any transaction.
+        /// Charged at the start of execution.
         min_transaction_gas_units: u64,
 
-        /// Any transaction over this size will be charged `INTRINSIC_GAS_PER_BYTE` per byte
+        /// Any transaction over this size will be charged `intrinsic_gas_per_byte` units of gas per byte.
```

Comment:
This comment (also in gas_schedule.rs) still isn't quite right. The new comment for `intrinsic_gas_per_byte` correctly explains how the cutoff works. I don't think it makes sense to repeat the formula in both comments. Can you find a way to say it once? Maybe: "Any transaction over this size will be charged an additional gas fee."?